### DISCUSSION TITLE
Quick picks error

### DISCUSSION THREAD

**Discussion by Ctrl-sza18** [fast4x/RiMusic#3778] [General]
Yo in my quick picks the monthly playlist and charts don't show on my phone it has been like this even when I switched phones 
![Screenshot_20240925_135657_it.fast4x.rimusic.jpg](https://github.com/user-attachments/assets/1735dae6-bfa3-4ffd-9a93-952c07ddb150)

**Comment by fast4x** [comment]
Check if enabled in settings

**Reply by Ctrl-sza18** [reply]
Still nothing it works on my mom's phone so I honestly don't know

**Reply by fast4x** [reply]
Android version?

**Reply by Ctrl-sza18** [reply]
Yep I updated to the new version and still nothing and the charts and monthly are on
